grapher/text/MarkdownTextWrap.tsx

```diff
@@ -556,13 +555,23 @@ export class MarkdownTextWrap extends React.Component<MarkdownTextWrapProps> {
         y: number,
         options?: React.SVGProps<SVGTextElement>
     ): JSX.Element | null {
-        const { lines } = this
+        const { lines, fontSize, lineHeight } = this
         if (lines.length === 0) return null
 
+        // Magic number set through experimentation.
+        // The HTML and SVG renderers need to position lines identically.
+        // This number was tweaked until the overlaid HTML and SVG outputs
+        // overlap.
+        const HEIGHT_CORRECTION_FACTOR = 0.74
```

Comment:
Copied this from `TextWrap`:

https://github.com/owid/owid-grapher/blob/fba2aa94ecd7d2d06bd9a0161bc6ea410ef68326/grapher/text/TextWrap.tsx#L201-L205

Maybe the magic number should be shared... (in order to do that, the base branch (`details-on-demand`) needs to get the latest file from master)